### DISCUSSION TITLE
Is there any way to use Garmin's 2FA?

### DISCUSSION THREAD

**Discussion by lowgatelabs** [philosowaffle/peloton-to-garmin#169] [Q&A]
I recently got a garmin dash cam, and it forces you to activate 2FA on your garmin account (which I use for Connect obviously). Is there any way to activate 2FA with p2g? Failing that it would seem my only other option would be creating a new account for the dash cam but I would prefer to keep everything garmin under the same banner.

Thanks!

**Comment by philosowaffle** [comment]
Unfortunately, no this is not something P2G can support. 

2FA cannot be handled by scripts like this, typically products that support 2FA also support a public API and OAuth for 3rd parties to use.  However, Garmin has chosen not to support a public API and instead will only provide this support to other large companies (not individual developers like myself).  

This is an extremely disappointing choice on Garmin's part, and greatly limits what the open source community can do.
